packages/twenty-server/src/modules/messaging/message-import-manager/drivers/gmail/services/gmail-get-message-list.service.ts

```diff
@@ -64,18 +66,23 @@ export class GmailGetMessageListService {
           };
         });
 
-      pageToken = messageList.data.nextPageToken ?? undefined;
-      hasMoreMessages = !!pageToken;
-
       const { messages } = messageList.data;
+      const hasMessages = messages && messages.length > 0;
```

Comment:
is it possible to add a test on this case?

Reply:
i will yes